docs/developer/devops/gitlab-ci-cd.md

```diff
@@ -0,0 +1,170 @@
+# GitLab CI/CD Infrastructure
+
+Given that files in the Library are stored in a Git repository, the approach
```

Comment:
Gitlab parent-child pipeline (with gitlab.com....ref) are used to trigger execution of digital twins stored in one gitlab repository.